src/pages/index.tsx

```diff
@@ -79,7 +79,7 @@ export default function Home(): JSX.Element {
                   <li><a href="/cache/develop/authentication"><Translate id="authenticationAndAccessControl">Authentication and access control</Translate></a></li>
                   <li><a href="/cache/develop/basics/error-handling-production-readiness"><Translate id="properlyHandlingErrorsInMomento">Properly handling errors in Momento</Translate></a></li>
                   <li>
-                    <a href="https://aws.amazon.com/marketplace/pp/prodview-ndqotnrhhjyvo" target="_blank">
```

Comment:
this was pointing to old deprecated/hidden listing so updated it to new one quick while was in here.